packages/cli/src/lib/snapshot/snapshotReporter.ts

```diff
@@ -57,6 +58,39 @@ export class SnapshotReporter {
     );
   }
 
+  public getReportStatus(): SnapshotReportStatus {
+    if (this.isSuccessReport()) {
+      return SnapshotReportStatus.SUCCESS;
+    }
+    if (this.isVaultEntriesMissingReport()) {
+      return SnapshotReportStatus.MISSING_VAULT_ENTRIES;
+    }
+    return SnapshotReportStatus.ERROR;
+  }
```

Comment:
Allow simpler handling for the caller.
However, maybe we should allow the caller to register callbacks for 'success'/'missing'/'error' and compute from here (given the refacto tagged on preview and push).

To be considered.